src/components/OMBInfo/OMBInfo.jsx

```diff
@@ -101,6 +101,7 @@ class OMBInfo extends React.Component {
           id="omb-modal"
           visible={this.state.modalOpen}
           onClose={this.closeModal}
+          title=""
```

Comment:
What is the impact of this?

Reply:
This was an artifact from manual testing, I'll take it out.